api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -45,6 +47,16 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String refAppLocationGlobalProperty = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME);
+        if(StringUtils.isNotBlank(refAppLocationGlobalProperty) && refAppLocationGlobalProperty != LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME) {
```

Comment:
String comparison is not usually done like that.

Reply:
Added fix @dkayiwa